src/azure-cli-core/azure/cli/core/tests/test_command_registration.py

```diff
@@ -298,10 +298,10 @@ def _check_index():
         loader.load_command_table(["hello", "mod-only"])
         _check_index()
 
-        with mock.patch("azure.cli.core.__version__", "2.7.0"), mock.patch.object(cli.cloud, "profile", "2019-03-01-hybrid"):
+        with  mock.patch.object(cli.cloud, "profile", "2019-03-01-hybrid"):
```

Comment:
Fix ` AttributeError: <MagicMock id='2694374960976'> does not have the attribute '__version__'` error.
In 3.11, mock can only mock selected magic method, `__version__` is not in mockable.
> The only exceptions are magic methods and attributes (those that have leading and trailing double underscores). Mock doesn’t create these but instead raises an [AttributeError](https://docs.python.org/3/library/exceptions.html#AttributeError). This is because the interpreter will often implicitly request these methods, and gets very confused to get a new Mock object when it expects a magic method. If you need magic method support see [magic methods](https://docs.python.org/3/library/unittest.mock.html#magic-methods).

Ref: https://docs.python.org/3/library/unittest.mock.html#id3